src/os/command/feature/featureopacitycmd.js

```diff
@@ -69,3 +70,21 @@ os.command.FeatureOpacity.prototype.applyValue = function(configs, value) {
 
   os.command.FeatureOpacity.base(this, 'applyValue', configs, value);
 };
+
+
+/**
+ * @inheritDoc
+ */
+os.command.FeatureOpacity.prototype.finish = function(configs) {
+  // dispatch the opacity change event on the source for the histogram
+  var feature = /** @type {ol.Feature} */ (this.getFeature());
+  var source = /** @type {plugin.file.kml.KMLSource} */ (os.feature.getSource(feature));
```

Comment:
Same comment about referencing the KML plugin here.

Reply:
Same comment as previous plugin reference.